src/main/webapp/index.html

```diff
@@ -3,6 +3,55 @@
 
 <title>ZoomTube</title>
 <link rel="stylesheet" href="/style.css" />
-<p>ZoomTube</p>
+<link
+  rel="stylesheet"
+  href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+  integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+  crossorigin="anonymous"
+/>
 
-<a href="/list.html">View list of lectures</a>
+<body>
+  <nav class="navbar navbar-light">
+    <h2 id="header-text">ZoomTube</h2>
+  </nav>
+  <div id="text">
+    <h1 class="display-6">Project Overview</h1>
+    <br />
+    <p>
+      ZoomTube is a web application that facilitates the viewing of recorded university lectures. With classes shifting
+      towards a virtual environment, lectures have become disengaging, especially for those watching asynchsronously.
+      Our web application aims to mitigate this issue by providing a way for students to interact with each other on a
+      single platform.
+    </p>
+    <br />
+    <p>
+      Recorded university lectures are synchronized with their transcript and a student discussion forum. Within this
```

Comment:
I feel these 2 sentences are a repetition of the first. The first one mentions the student discussion forum, so I am not sure what these sentences add in terms of information.

Reply:
I think both of these sentences are crucial, the first explains the high level synchronization feature while the next goes into how students would use this product.